<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DTS rev. C control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_shift1">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_shift1">
      <property name="text">
       <string>Shift 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_gain1">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_gain1">
      <property name="text">
       <string>Gain 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_slope1">
      <property name="text">
       <string>Slope 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_slope1">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_shift2">
      <property name="text">
       <string>Shift 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_gain2">
      <property name="text">
       <string>Gain 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_slope2">
      <property name="text">
       <string>Slope 2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="spinBox_shift2">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="spinBox_gain2">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="spinBox_slope2">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBox_hv1">
      <property name="maximum">
       <number>4095</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="spinBox_hv2">
      <property name="maximum">
       <number>4095</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_hv1">
      <property name="text">
       <string>HV 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_hv2">
      <property name="text">
       <string>HV 2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox_fol">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_fol">
      <property name="text">
       <string>FOL current</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_dil">
      <property name="text">
       <string>DIL pulse amplitude</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_dil">
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="checkBox_pd1">
      <property name="text">
       <string>Pulse duration 1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="checkBox_pd2">
      <property name="text">
       <string>Pulse duration 2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="checkBox_led1">
      <property name="text">
       <string>LED 1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="checkBox_led2">
      <property name="text">
       <string>LED 2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_status_static">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_status">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_t_static">
      <property name="text">
       <string>Temperature:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_t">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
